TransactionReporting/get-account-updater-job-details.py

```diff
@@ -0,0 +1,100 @@
+    paging.limit = 1000
+    paging.offset = 2
+    request = apicontractsv1.getAUJobDetailsRequest()
+    request.merchantAuthentication = merchantAuth
+    request.paging = paging
+    request.month = "2018-08"
+    request.modifiedTypeFilter = "all"
+    request.refId = "123456"
+    controller = getAUJobDetailsController(request)
+    controller.execute()
+    response = controller.getresponse()
+
+    if response is not None:
+        if response.messages.resultCode == apicontractsv1.messageTypeEnum.Ok:
+
+                if hasattr(response, 'auDetails'):
+                    print('SUCCESS: Get Account Updater job details for Month and year :' + request.month)
+                    if response.messages is not None:
+                        print('Message Code: %s' % response.messages.message[0]['code'].text)
+                        print('Message Text: %s' % response.messages.message[0]['text'].text)
+                        print('Total Number In Results: %s' % response.totalNumInResultSet)
+                        print('\n')
+
+                        for details in response.auDetails.auDelete:
+                            print('Deleted Profile:')
+                            # auDelete Start
+                            print('Customer Payment Profile ID: %s' % details.customerPaymentProfileID)
+                            print('Customer  Profile ID: %s' % details.customerProfileID)
+                            print('First Name: %s' % details.firstName)
+                            print('Last Name: %s' % details.lastName)
+                            print('AU Reason Code: %s' % details.auReasonCode)
+                            print('Reason Description: %s' % details.reasonDescription)
+                            print('Update Time UTC: %s' % details.updateTimeUTC)
+                            print(' ')
+                            # fetching card details:
+                            print('Card Details:')
+                            print('Card Number: %s' % details.creditCard.cardNumber)
+                            print('Card Type: %s' % details.creditCard.cardType)
+                            print('Expiration Date: %s' % details.creditCard.expirationDate)
+                            # fetching deleted subscriptions
+                            if hasattr(details.subscriptionIdList, 'subscriptionId') == True:
+                                print(' ')
+                                print('Subscription Id: %s' % details.subscriptionIdList.subscriptionId)
+                            # auDelete End
+                        print('\n')
+
+                        for details in response.auDetails.auUpdate:
+
+                                # auUpdate Start
+                                print('Updated Profile:')
+                                print('Customer Payment Profile ID: %s' % details.customerPaymentProfileID)
+                                print('Customer  Profile ID: %s' % details.customerProfileID)
```

Comment:
Remove extra space in "Customer Profile ID". Also Customer Profile ID should come before Customer Payment Profile ID.

Reply:
Removed extra space in "Customer Profile ID" and placed  Customer Profile ID before Customer Payment Profile ID.